gtda/homology/rescaling.py

```diff
@@ -163,12 +172,162 @@ def transform(self, X, y=None):
             after consistent rescaling.
 
         """
-        # Check if fit had been called
-        check_is_fitted(self, '_is_fitted')
+        check_is_fitted(self)
         X = check_array(X, allow_nd=True)
 
         Xt = Parallel(n_jobs=self.n_jobs)(
             delayed(self._consistent_homology_distance)(X[i])
             for i in range(X.shape[0]))
         Xt = np.array(Xt)
         return Xt
+
+
+@adapt_fit_transform_docs
+class ConsecutiveRescaling(BaseEstimator, TransformerMixin):
+    """Rescaling of distances between consecutive pairs of points by a fixed
+    factor.
+
+    The computation during :meth:`transform` depends on the nature of the array
+    `X`. If each entry in `X` along axis 0 represents a distance matrix
+    :math:`D`, then the corresponding entry in the transformed array is the
+    distance matrix :math:`D'_{ii+1} = factor \times D_{ii+1}`. If the entries
```

Comment:
... distance matrix :math:`D'`, where :math:`D'_{ii+1} = factor \times D_{ii+1}`

Reply:
I also think that the subscripts will look confusing here in LaTeX.  Perhaps we want D'_{i,i+1} etc? We might have to change this is in `ConsistentRescaling` for consistency (no pun intended).